### PR TITLE
[Venom] Security test random mac and other async fixes

### PR DESCRIPTION
**Pull request by JeGoi**
# Description
Add test with random mac address security event with inline l2

# Impacts
Do the test in venom. Can be removed from testlink

# Delete branch after merge
YES

## New Features
* test security random mac address with inline l2

### CODE REVIEW

**Comment by JeGoi** [comment]
https://gitlab.com/inverse-inc/packetfence/-/pipelines/671540811

**Comment by JeGoi** [comment]
New pipeline looks fine: https://gitlab.com/inverse-inc/packetfence/-/pipelines/683505770

File: t/venom/test_suites/security_event_random_mac/05_setup_packetfence.yml [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+    headers:
+      "Authorization": "{{.get_login_token.result.token}}"
+      "Content-Type": "application/json"
+    assertions:
+      - result.statuscode ShouldEqual 200
+
+- name: restart_iptables
+  steps:
+  - type: systemctl_service_restart
+    service: packetfence-iptables
```
Comment (nqb) [inline t/venom/test_suites/security_event_random_mac/05_setup_packetfence.yml]: All services below will need to be restarted through API to be consistent with changes made on #7367

File: t/venom/test_suites/security_event_random_mac/30_tests.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+       "mac": "{{.security_event_random_mac.node.macaddress}}"
+     }
+    headers:
+      "Authorization": "{{.get_login_token.result.token}}"
+      "Content-Type": "application/json"
+    assertions:
+      - result.statuscode ShouldEqual 200
+
+
+- name: test_if_the_device_is_back_in_the_register_ipset
```
Comment (nqb) [inline t/venom/test_suites/security_event_random_mac/30_tests.yml]:
```suggestion
- name: test_if_the_device_is_back_in_the_unregister_ipset
```

File: t/venom/test_suites/security_event_random_mac/teardown/05_deconfigure_packetfence.yml [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+       "id": "inline",
+       "interfaceSNAT": ""
+     }
+    headers:
+      "Authorization": "{{.get_login_token.result.token}}"
+      "Content-Type": "application/json"
+    assertions:
+      - result.statuscode ShouldEqual 200
+
+- name: desable_security_event
```
Comment (nqb) [inline t/venom/test_suites/security_event_random_mac/teardown/05_deconfigure_packetfence.yml]:
```suggestion
- name: disable_security_event
```

File: t/venom/test_suites/security_event_random_mac/teardown/05_deconfigure_packetfence.yml [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+  steps:
+  - type: pf_api_action
+    method: PATCH
+    url: 'config/security_event/{{.security_event_random_mac.event.id}}'
+    body: '{"id":"{{.security_event_random_mac.event.id}}","enabled":"N","quiet":true}'
+
+
+- name: restart_iptables
```
Comment (nqb) [inline t/venom/test_suites/security_event_random_mac/teardown/05_deconfigure_packetfence.yml:79]: All services below will need to be restarted through API to be consistent with changes made on https://github.com/inverse-inc/packetfence/pull/7367

**Review by nqb — CHANGES_REQUESTED** [review]

File: t/venom/test_suites/common/restart_pfdns_service.yml [hunk 5]
```diff
@@ -1,17 +1,6 @@
 name: Restart pfdns service
```
Comment (nqb) [inline t/venom/test_suites/common/restart_pfdns_service.yml:1]: We have two files which do same thing:
- restart_pfdns.yml
- restart_pfdns_service.yml

Comment (nqb) [reply t/venom/test_suites/common/restart_pfdns_service.yml:1]: I removed restart_pfdns.yml to be consistent with current naming convention.

**Pull request merged**